src/app/authoring-tool/edit-component-advanced/edit-component-advanced.component.html

```diff
@@ -0,0 +1,107 @@
+      <edit-discussion-advanced
+        *ngSwitchCase="'Discussion'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-discussion-advanced>
+      <edit-draw-advanced
+        *ngSwitchCase="'Draw'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-draw-advanced>
+      <edit-embedded-advanced
+        *ngSwitchCase="'Embedded'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-embedded-advanced>
+      <edit-graph-advanced
+        *ngSwitchCase="'Graph'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-graph-advanced>
+      <edit-dialog-guidance-advanced
+        *ngSwitchCase="'DialogGuidance'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-dialog-guidance-advanced>
+      <edit-html-advanced
+        *ngSwitchCase="'HTML'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-html-advanced>
+      <edit-label-advanced
+        *ngSwitchCase="'Label'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-label-advanced>
+      <edit-match-advanced
+        *ngSwitchCase="'Match'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-match-advanced>
+      <edit-multiple-choice-advanced
+        *ngSwitchCase="'MultipleChoice'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-multiple-choice-advanced>
+      <edit-open-response-advanced
+        *ngSwitchCase="'OpenResponse'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-open-response-advanced>
+      <edit-outside-url-advanced
+        *ngSwitchCase="'OutsideURL'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-outside-url-advanced>
+      <edit-peer-chat-advanced
+        *ngSwitchCase="'PeerChat'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-peer-chat-advanced>
+      <ng-container
+        *ngSwitchCase="
+          component.content.type === 'ShowGroupWork' || component.content.type === 'ShowMyWork'
+            ? component.content.type
+            : ''
+        "
+      >
+        <edit-component-constraints
+          [componentContent]="component.content"
+        ></edit-component-constraints>
+        <edit-component-json [component]="component"></edit-component-json>
+      </ng-container>
+      <edit-summary-advanced
+        *ngSwitchCase="'Summary'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-summary-advanced>
+      <edit-table-advanced
+        *ngSwitchCase="'Table'"
+        [nodeId]="nodeId"
+        [componentId]="component.id"
+      ></edit-table-advanced>
+    </ng-container>
+  </div>
+</div>
+<div mat-dialog-actions fxLayout="row" fxLayoutAlign="end center">
+  <button mat-button color="primary" (click)="close()" i18n>Done</button>
+</div>
```

Comment:
Make this similar to preview-component-dialog.component.html?
```suggestion
<mat-divider></mat-divider>
<mat-dialog-actions fxLayoutAlign="end">
  <button mat-button mat-dialog-close cdkFocusRegionstart i18n>Close</button>
</mat-dialog-actions>
```